device/alpaka/include/traccc/alpaka/seeding/track_params_estimation.hpp

```diff
@@ -30,7 +31,7 @@ struct track_params_estimation
     /// @param copy The copy object to use for copying data between device
     ///             and host memory blocks
     track_params_estimation(const traccc::memory_resource& mr,
-                            vecmem::copy& copy);
+                            traccc::alpaka::vecmem::device_copy& copy);
```

Comment:
The member may have changed, but the interface still expects a concrete type. This should also stay as-is. (Adding a `::` prefix to the `vecmem` namespace may be needed here as well though. Or you could avoid introducing a `traccc::alpaka::vecmem` namespace. 🤔)